packages/types/src/interview.ts

```diff
@@ -22,9 +22,12 @@ export type Self = {
      * The interviewee id (the id of the tutor)
      */
     interviewee: number;
-    call: number;
     rule: number;
+    session: ISession.Id; /** @deprecated */
```

Comment:
No need to mark it as deprecated. 


```suggestion
    session: ISession.Id;
```